Binary files bible/.DS_Store and biblepatched/.DS_Store differ
diff -ruN bible/Makefile biblepatched/Makefile
--- bible/Makefile	Wed Dec 14 20:53:00 1994
+++ biblepatched/Makefile	Mon Dec 27 11:10:24 2004
@@ -46,11 +46,11 @@
 NINSTPKG      = $(PROGRAM).pkg
 
 # where to install this stuff
-DEST	      = /usr
+DEST	      = @FINKPREFIX@
 #DEST	      = $(HOME)/local
 DESTBIN	      = $(DEST)/bin
 DESTLIB       = $(DEST)/lib
-DESTMAN	      = $(DEST)/man
+DESTMAN	      = $(DEST)/share/man
 DESTMAN1      = $(DESTMAN)/man1
 
 # I use environment variables for these...
@@ -405,23 +405,17 @@
 		@if [ $(DESTBIN) != . ]; then \
 		((cd $(DESTBIN);rm -f $(PROGRAM)); \
 		cp $(PROGRAM) $(DESTBIN)); \
-		chown bin $(DESTBIN)/$(PROGRAM); \
-		chgrp bin $(DESTBIN)/$(PROGRAM); \
 		chmod 755 $(DESTBIN)/$(PROGRAM); fi
 		@echo Installing $(DATA) and $(DATA).conc in $(DESTLIB)
 		@if [ ! -r $(DESTLIB) ]; then mkdir $(DESTLIB); fi
 		@if [ $(DESTLIB) != . ]; then \
 		cp $(DATA) $(DATA).conc $(DESTLIB); \
-		chown bin $(DESTLIB)/$(DATA) $(DESTLIB)/$(DATA).conc; \
-		chgrp bin $(DESTLIB)/$(DATA) $(DESTLIB)/$(DATA).conc; \
 		chmod 644 $(DESTLIB)/$(DATA) $(DESTLIB)/$(DATA).conc; fi
 		@if [ ! -r $(DESTMAN) ]; then mkdir $(DESTMAN); fi
 		@echo Installing $(MAN) in $(DESTMAN1)
 		@if [ ! -r $(DESTMAN1) ]; then mkdir $(DESTMAN1); fi
 		@if [ $(DESTMAN1) != . ]; then \
 		cp $(MAN) $(DESTMAN1)/$(MAN); \
-		chown bin $(DESTMAN1)/$(MAN); \
-		chgrp bin $(DESTMAN1)/$(MAN); \
 		chmod 644 $(DESTMAN1)/$(MAN); fi
 
 uninstall:;	@echo Un-installing $(PROGRAM), $(DATA) and $(MAN)
